Dockerfile

```diff
@@ -18,7 +23,8 @@ CMD ["make", "test"]
 FROM gcr.io/distroless/base-debian10 as vvgo
 COPY ./infra/vvgo/etc/mime.types /etc/
 COPY ./public /public
-COPY --from=builder vvgo vvgo
+COPY --from=builder vvgo /vvgo
+COPY --from=node node_modules /node_modules
```

Comment:
```suggestion
```

Reply:
We shouldn't be including `node_modules` in our final Docker image.

Reply:
Why cant we include them in the final image?